interp/interp.go

```diff
@@ -181,6 +181,7 @@ type opt struct {
 	stdin        io.Reader     // standard input
 	stdout       io.Writer     // standard output
 	stderr       io.Writer     // standard error
+	args         []string      // cmdline args
```

Comment:
The addition of `args` to `opt` is a completely different feature. It should not be mixed in the same PR and should submitted in a separate one. Thanks.